src/Test/Utilities/Portable/Diagnostics/DiagnosticDescription.cs

```diff
@@ -219,13 +217,8 @@ public override bool Equals(object obj)
                 {
                     if (_startPosition.Value != d._startPosition.Value)
                     {
-                        _showPosition = true;
-                        d._showPosition = true;
```

Comment:
Let us forever burn this mistake into our memories. Today will be remembered as ""In No State chANging Equals" day, or "INSANE" for short.